index.html

```diff
@@ -315,5 +315,58 @@ <h5 class="about-me-heading">About Me</h5>
         </ul>
       </div>
     </section>
+
+    <footer class="footer">
+      <h2 class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xpzbopvv" method="POST">
+        <ul class="input-ul">
+          <li class="form-items">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="name"
+              maxlength="30"
+              placeholder="Full name"
+              required
+            />
+          </li>
+          <li class="form-items">
+            <input type="email" placeholder="Email address" required />
+          </li>
+          <li class="form-textarea">
+            <textarea
+              name="textarea"
+              id="textarea"
+              maxlength="500"
+              placeholder="Enter your text here"
+              required
+            >
+            </textarea>
+          </li>
+          <li>
+            <button type="button" class="get-in-touch">Get in touch</button>
+          </li>
+        </ul>
```

Comment:
- [ ] Please remove the `ul tag` and the `li` from the form `input` for a good HTML semantic practice